include/genericCMPClient.h

```diff
@@ -23,14 +23,17 @@ extern "C" {
 
 # include <openssl/opensslv.h>
 
+#define OPENSSL_3_2_FEATURES (OPENSSL_VERSION_NUMBER >  0x30200000L || defined(USE_LIBCMP))
+#define OPENSSL_3_3_FEATURES (OPENSSL_VERSION_NUMBER >  0x30300000L || defined(USE_LIBCMP))
```

Comment:
Should it be ">=" instead of only ">"? 
or macro name need to be updated.

Reply:
I had thought that normally, the `>` fits better (because new CMP features where contributed to OpenSSL during the development/pre-release phase) while I experienced that it does not work well at least for 3.4.0.
Looks like there is an inconsistency in OpenSSL on the use of `OPENSSL_VERSION_PRE_RELEASE` in `opensslv.h` .
So likely we are more in the safe side to use `>=` also for 3.2 and 3.3 - changed the definitions accordingly.